cesm_tools.py

```diff
@@ -221,6 +223,18 @@ def setup_cesm_explicit(
             cyclic_x=cyclic_x,
         )
 
+        # Load Balancing Math
+        total_number_of_points = nx * ny
+        nodes=1
+        pts_per_processor = total_number_of_points/ float(processors_per_node)
+        while not pts_per_processor < ideal_number_of_points_per_core_ceiling:
```

Comment:
How about `while pts_per_processor > ideal_number_of_points_per_core_ceiling`? We probably are okay with the edge case of `pts_per_processor == ideal_number_of_points_per_core_ceiling` and don't need to enforce it to be strictly less than.

Reply:
Sounds good!